pkg/cli/zip_table_registry.go

```diff
@@ -1100,12 +1100,9 @@ var zipSystemTables = DebugZipTableRegistry{
 			FROM system.job_info`,
 	},
 	"system.lease": {
-		nonSensitiveCols: NonSensitiveColumns{
-			`"descID"`,
-			"version",
-			`"nodeID"`,
-			"expiration",
-		},
+		customQueryRedacted: `
+			SELECT * FROM system.lease;
+		`,
```

Comment:
> 		SELECT * FROM system.lease;

nit: I'm a bit concerned that using `SELECT *` here could cause us to leak newly added sensitive columns into redacted debug zip bundles. Any reason why we can't explicitly define the non-sensitive columns?